src/components/Badge/Badge.vue

```diff
@@ -10,6 +10,7 @@
                { 'bg-coral-100 text-error': error },
                { 'bg-turquoise-100 text-turquoise-900': turquoise },
                { 'bg-flint-100 text-gray-700': flint },
+               { 'bg-gradient-114 from-[#1876db] to-[#5748ff] hover:from-[#5748ff] hover:to-[#1876db] text-white': gradient },
```

Comment:
@bethqiang I am not sure if this should change color on hover? 
I'd vote no, because
that's the button behavior to hint to the user that it is clickable, but it won't be clickable as a tooltip, right?

Reply:
good point but it's using the same colors, right? I tried hovering over it on storybook and it doesn't seem to change colors.

Reply:
did you go on the preview link? it does change for me

https://user-images.githubusercontent.com/50080618/192861399-f319a5f9-7f0e-4249-b1d1-2d0c008299e2.mov

Reply:
> did you go on the preview link?
obviously, as the `gradient` prop didn't exist before :)

> it does change for me
oh it is so subtle I didn't notice it, you are right

Reply:
oooh good point. snuck in when I copy pastaed from the button. will change!

Reply:
done!